pages/quiz/[question].js

```diff
@@ -1,9 +1,22 @@
+import Image from 'next/image';
 import Link from 'next/link';
+import { useRouter } from 'next/router';
 import styled from 'styled-components';
-import Image from 'next/image';
+import { quiz } from '../../lib/quiz';
 import header_picture from '/public/header-picture.jpeg';
 
-export default function Foodlist() {
+const Question = () => {
+  const router = useRouter();
+  const { question } = router.query;
+
+  const currentQuestion = quiz[question];
+
+  if (!currentQuestion) {
+    return <div>Question not found :(</div>;
+  }
+
+  console.log(currentQuestion);
```

Comment:
please remove console.log before merging

Reply:
done, thanks for the review and hint!